W-17772543-SFTP-2-4-2-Write-operation-custom-mode-modifications (#2788)
mulesoft/docs-connectors: sftp/2.4/modules/ROOT/pages/sftp-write.adoc

Additionally, you can configure the buffer size for writing into the file via the *Buffer size for write strategy* field. 

[NOTE]

> Use the `CUSTOM` write strategy only in `CUSTOM_APPEND` mode as it is not applicable to the `OVERWRITE` or `CREATE_NEW` modes. Use this strategy only if the `STANDARD` strategy results in errors during file writing in `APPEND` mode. Concurrency is not maintained for servers that do not follow SFTP guidelines.

Reviewer: ```suggestion
Use the `CUSTOM` write strategy with the `CUSTOM_APPEND` mode because it isn't applicable to the `OVERWRITE` or `CREATE_NEW` modes. Use this strategy only if the `STANDARD` strategy results in errors during file writing in `APPEND` mode. Concurrency isn't maintained for servers that don't follow SFTP guidelines.
```
